src/playground/src/components/playground/ChatParamsCard.tsx

```diff
@@ -77,7 +77,7 @@ export const ChatParamsCard = ({
         <ParamSelect
           label="Temperature"
           explain="Controls randomness. Lowering the temperature means that the model will produce more repetitive and deterministic responses. Increasing the temperature will result in more unexpected or creative responses. Try adjusting temperature or Top P but not both."
-          onUpdate={(newValue) => tokenUpdate("temperature", newValue)}
+          onUpdate={(newValue) => tokenUpdate("temperature", Number(newValue))}
```

Comment:
Don't use the `Number` constructor as that creates an object of type `Number` rather than a number. If we need it to be numerical use `parseInt`